gradle/gradle-mvn-push.gradle

```diff
@@ -11,4 +11,21 @@ publishing {
             url = "${rootProject.buildDir}/localMaven"
         }
     }
+
+    /**
+     * Want to push to an internal repository for testing?
+     * Set the following properties in ~/.gradle/gradle.properties.
+     *
+     * internalUrl=YOUR_INTERNAL_URL
+     * internalUsername=YOUR_USERNAME
+     * internalPassword=YOUR_PASSWORD
+     */
+    def internalUrl = providers.gradleProperty("internalUrl")
+    if (internalUrl.isPresent()) {
+        maven {
+            name = "internal"
+            url = uri(internalUrl)
+            credentials(PasswordCredentials)
+        }
+    }
```

Comment:
I've not tested this. I couldn't do a proper copy-paste as the code block in Wire is in Kotlin Script, so I had to rewrite it in Groovy. I could do another internal release to test this out, but the new release would be identical to the last one.

Reply:
This is how Paparazzi does it!
https://github.com/cashapp/paparazzi/blob/master/paparazzi/build.gradle#L59-L72